store/resource.js

```diff
@@ -10,6 +10,8 @@ class Resource {
 
   static defaultValues = {}
 
+  @observable error = null
```

Comment:
Why do you need thus property?

Reply:
I don't but I thought this may be good if the method remains consistent, i.e. in the success scenario it stores some properties into `this`, so in the error scenario it stores the error into `this`.

Also, I think that the information itself may be useful too to show some error reporting in the future.

What do you think about?

Reply:
I try to follow my personal rule - everything which is not used shouldn't be in the repository.  It just makes the other developers think why is it there.  There has to be some reason and if it gets remove something will break.